hcert_spec.md

```diff
@@ -246,15 +246,25 @@ While patterned on best practices of the ICAO ML, there are a number of simplifi
 
 In order to alleviate the burden of countries during the initial phase, there shall be a secretarial service which will:
 
-* Maintain a list of operational and legal contacts for each Member State to further the orderly management of this health specific set of master lists.
-* Maintain a public 24x7 incident/security contact point.
+### version 1.00 - secretariat tasks
+
+In a later version - the service may also:
+
 * Maintain a public list of URLs with the most up to date CSCA lists for each Member State. (DGCG)
```

Comment:
I would consider dropping the URL for CSCA list for each Member state here but keep the aggregated list of CSCA Certificates below. Since CSCA Certificates are self signed it is of little value to have them just downloadable by a URL without properly defined integrity mechanisms that is beyond MS capacity in version 1.00. I suspect that CSCA must be provided to the Secretariat by OOB means (or securely uploaded) at least in version 1.00

Reply:
Dropped & moved.

Reply:
OK. That took care of my concern.